src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/KafkaTransportContext.cs

```diff
@@ -0,0 +1,129 @@
+namespace MassTransit.KafkaIntegration
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Confluent.Kafka;
+    using MassTransit.Configuration;
+    using Serializers;
+    using Transports;
+
+
+    public class KafkaTransportContext<TKey, TValue> :
```

Comment:
TopicKafkaSendTransportContext, or something that isn't empty of send or receive would be good.